scripts/create-package/plopfile.ts

```diff
@@ -56,41 +57,34 @@ module.exports = (plop: NodePlopAPI) => {
       },
       {
         type: 'confirm',
-        name: 'hasExamples',
-        message: 'Create example scaffolding?',
+        name: 'isConverged',
+        message: 'Is this a converged package?',
         default: true,
-        when: answers => answers.target === 'react',
       },
       {
         type: 'confirm',
         name: 'publish',
-        message: answers =>
-          answers.hasExamples
-            ? 'Should the package be published right away (must be Yes for packages with example scaffolding)?'
-            : 'Should the package be published right away?',
-        default: (answers: Answers) => !!answers.hasExamples,
+        message: 'Should the package be published right away?',
```

Comment:
perhaps let's just remove this and always set `private` at the worst case you'll realize the next day you didn't release what you wanted to and just rerelease